docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -442,10 +302,129 @@ This file is picked up when the server starts as indicated by the following AUDI
+
+To use the proxy datasource:
+1. Set the `fhirServer/persistence/jdbc/enableProxyDatasource` flag to true (default is false):
+    ```
+        {
+            "fhirServer": {
+                …
+                "persistence": {
+                    …
+                    "jdbc": {
+                        …
+                        "enableProxyDatasource": true
+                    }
+                    …
+                }
+        }
+    ```
+2. Set the `fhirServer/persistence/jdbc/dataSourceJndiName` to `jdbc/fhirProxyDataSource`.
+3. Collect the fhir-persistence-proxy jar and all of its dependencies (including fhir-core and fhir-config) under `<WLP_HOME>/usr/shared/resources` and define a corresponding shared resource library in the liberty config.
+4. Configure the datasource definition in a server.xml configDropin (for example, by moving `configDropins/disabled/datasource-proxy.xml` to `configDropins/overrides`). The datasource definition should look like this:
+    ```
+    <dataSource id="fhirProxyDataSource" jndiName="jdbc/fhirProxyDataSource" type="javax.sql.XADataSource">
+        <jdbcDriver libraryRef="fhirSharedLib" javax.sql.XADataSource="com.ibm.fhir.persistence.proxy.FHIRProxyXADataSource" />
+    </dataSource>
+    ```
+5. Add `connectionProperties` to each datasource definition within the tenant fhir-server-config.json files.
+    Here is an example depicting a Db2 datasource definition:
+    ```
+    {
+        "fhirServer":{
+            "persistence":{
+                "factoryClassname":"com.ibm.fhir.persistence.jdbc.FHIRPersistenceJDBCFactory",
+                "datasources": {
+                    "study1": {
+                        "tenantKey": "<the-tenant-key>",
+                        "type": "db2",
+                        "connectionProperties": {
+                            "serverName": "mydb2server",
+                            "portNumber": 50000,
+                            "user": "db2inst1",
+                            "password": "********",
+                            "databaseName": "FHIRDB",
+                            "currentSchema": "FHIR1",
+                            "driverType": 4
+                        }
+                    }
+                }
+            }
+        }
+    }
+    ```
+
+    The `connectionProperties` property is a set of driver-specific properties needed to connect to an instance of that database type. For a Db2-related datasource definition, any bean property supported by the `DB2XADataSource` class can be specified within the `connectionProperties` property group. For a discussion of the specific properties that can be used to configure a `DB2XADataSource` instance, see the [Db2 Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSEPGG_11.1.0/com.ibm.db2.luw.apdv.java.doc/src/tpc/imjcc_rjvdsprp.html).
```

Comment:
"any bean property"

Reply:
I didn't write it, but I agree with your take:
```suggestion
    The `connectionProperties` property is a set of driver-specific properties needed to connect to an instance of that database type. For a Db2, any datasource property supported by the `DB2XADataSource` class can be specified within the `connectionProperties` property group. For a discussion of the specific properties that can be used to configure a `DB2XADataSource` instance, see the [Db2 Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSEPGG_11.1.0/com.ibm.db2.luw.apdv.java.doc/src/tpc/imjcc_rjvdsprp.html).
```